ui/web-v2/src/proto/notification/service_pb.d.ts

```diff
@@ -779,6 +781,9 @@ export class ListSubscriptionsRequest extends jspb.Message {
   getEnvironmentId(): string;
   setEnvironmentId(value: string): void;
 
+  getOrganizationId(): string;
+  setOrganizationId(value: string): void;
```

Comment:
You must configure the organization ID on the current console, too.

https://github.com/bucketeer-io/bucketeer/pull/1373/files#diff-af6afe85654f2d43e9867c26ff6e6d6731432dec50cd3a8b4b11065566fa0631R138

Reply:
I have configured the organization ID, can you check again?